configure

```diff
@@ -15852,40 +15840,11 @@ case "${host_os}" in
 
            *)
                                                                                     if xcodebuild -version -sdk macosx10.5 Path >/dev/null 2>&1 ; then
-                 mac_version_min="-mmacosx-version-min=10.3"
+                 mac_version_min="-mmacosx-version-min=10.5"
```

Comment:
Did you mean 10.6?

Reply:
Not if the SDK is 10.5

Reply:
This file is auto generated.

Reply:
ah, thanks for clarifying. I generally recommend against putting autogenerated files into version control